<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoverCreatorMainWindow</class>
 <widget class="QMainWindow" name="MoverCreatorMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1900</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MoverCreator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="Designer::MoverCreatorWidget" name="mMoverCreatorWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="mRevertAction">
   <property name="text">
    <string>Revert</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionEngine_Designer">
   <property name="text">
    <string>Engine Designer</string>
   </property>
  </action>
  <action name="actionWeapon_Designer">
   <property name="text">
    <string>Weapon Designer</string>
   </property>
  </action>
  <action name="actionWeapon_Designer_New">
   <property name="text">
    <string>Weapon Designer New</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Designer::MoverCreatorWidget</class>
   <extends>QWidget</extends>
   <header>MoverCreatorWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/movercreator.qrc"/>
 </resources>
 <connections/>
</ui>
